include/deal.II/base/parameter_handler.h

```diff
@@ -963,6 +963,15 @@ class ParameterHandler : public Subscriptor
    */
   virtual void parse_input_from_xml (std::istream &input);
 
+  /**
+   * Parse input from an JSON stream to populate known parameter fields. This
+   * could be from a file originally written by the print_parameters() function
+   * using the XML output style and then modified by hand as necessary, or from
+   * a file written using this method and then modified by the graphical
+   * parameter GUI (see the general documentation of this class).
```

Comment:
Take out this sentence. The GUI doesn't use JSON (though it could, in principle).